s3-test/src/main/java/com/hazelcast/jet/tests/s3/S3WordCountTest.java

```diff
@@ -85,9 +90,14 @@ protected void init() {
     @Override
     protected void test() {
         long begin = System.currentTimeMillis();
+        int jobNumber = 0;
         while ((System.currentTimeMillis() - begin) < durationInMillis) {
-            jet.newJob(pipeline()).join();
-            verify();
+            JobConfig jobConfig = new JobConfig();
+            jobConfig.setName("s3-test-" + jobNumber);
+            jet.newJob(pipeline(), jobConfig).join();
+            verify(jobNumber);
+            logger.info(String.format("Job %d finished", jobNumber));
```

Comment:
Do we need this log on info level? Other logging seems ok since they happen only on some circumstances but this one will probably spam the log, WDYT?